### ISSUE TITLE
FRITZ!Box routers doesn't trust SSL.com ECC root certificate

### ISSUE THREAD

**Issue by JensSpanier** [dns-sb/DoT#8]
It seems like AVM FRTIZ!Box routers don't trust the root certificate "SSL.com Root Certification Authority ECC" (0x75e6dfcbc1685ba8).
It's only possible to use `dot.sb` if the certificate check is switched off.

I already contacted AVM and they replied
> I have just forwarded your request to our product management as a suggestion for improvement, so that the root certificate you mentioned will be implemented in an upcoming update.

Just wanted to let you know, that `dot.sb` can't currently be used with FRITZ!Box routers if the certificate check isn't switched off.

**Comment by Showfom** [comment]
We don't have that router for testing. Please request them to update their CA certificates.

**Comment by dpunkturban** [comment]
I have the same problem with the newest FRITZ!BOX (8.00) versions. `dot.sb` is not working on different routers.

**Comment by trkoch** [comment]
Confirmed with 8.03.

**Comment by Showfom** [comment]
> Confirmed with 8.03.

Please report the issue to the vendor, they should update their CA certificates.

Currently SSL.com is the only option to issue certificates for IPv6 address, we don't have other options :(

**Comment by JensSpanier** [comment]
Maybe this is worth a look for the future: https://letsencrypt.org/2025/07/01/issuing-our-first-ip-address-certificate/

**Comment by Showfom** [comment]
> Maybe this is worth a look for the future: https://letsencrypt.org/2025/07/01/issuing-our-first-ip-address-certificate/

Yes we will try to use Let's Encrypt when they have IP address certificate in public.

**Comment by trkoch** [comment]
> Please report the issue to the vendor, they should update their CA certificates.

I reported the issue to AVM and referenced this issue.

**Comment by JanST123** [comment]
Confirmed with FRITZ!OS:[7.60]

**Comment by Showfom** [comment]
We have migrated our SSL certificates from SSL.com to Let's Encrypt, please test if it's working now.

**Comment by ffPjrZUGXfcxuAj** [comment]
for me now work

<img width="583" height="55" alt="Image" src="https://github.com/user-attachments/assets/1db4715d-b378-4179-ac76-501902229bb8" />

**Comment by Showfom** [comment]
@ffPjrZUGXfcxuAj Thanks for your feedback, closing this issue.

**Comment by creopard** [comment]
@JensSpanier in an upcoming firmware version, FritzBoxes will be able to use "ssl.com" root certificate.
This issue was also affecting Cloudflare's DNS servers if you wanted to use them with DoT. 

excerpt from beta firmware (aka "Labor" version):
```
# Improvements of FRITZ!OS 8.24-129020 
## Internet:
- **Changed** Root certificate added for DNS over TLS with SSL.com
```
